src/Symfony/Component/HttpKernel/DataCollector/DataCollector.php

```diff
@@ -40,7 +38,7 @@
     /**
      * @var ClonerInterface
      */
-    private $cloner;
+    private static $cloner;
```

Comment:
because there is no state in it, and each datacollector is currently instanciating its own cloner, which I measured as a free-to-save overhead (addCasters being the "heavy" loop that takes some time).